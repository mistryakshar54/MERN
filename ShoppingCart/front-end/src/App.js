import React from 'react';
import AppHeader from "./components/Layout/Header/Header";
import AppContent from "./components/Layout/Content/Content";
import './App.scss';
import store from "./store/store";
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
function App() {
  return (
    <Provider store={store} >
      <div className="App">
        <Router>
          <AppHeader />
          <AppContent />
        </Router>
      </div>
    </Provider>
  );
}

export default App;
