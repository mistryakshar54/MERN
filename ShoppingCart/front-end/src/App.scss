@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');

.App {
  text-align: center;
  font-family: 'Roboto', sans-serif;
  .h1, .h2, .h3, .h4, .h5, .h6, h1, h2, h3, h4, h5, h6
  {
    font-family: 'Roboto', sans-serif;
  }
  .header-text-content
  {
    font-weight: 700;
  }
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 40vmin;
  pointer-events: none;
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}
