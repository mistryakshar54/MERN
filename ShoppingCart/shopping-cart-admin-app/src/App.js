import React from "react";
import './App.css';
import MainContent from './Components/AppLayout/MainContent/MainContent';
import store from "./store/index";
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
function App() {
  return (
    <Provider store={store}>
      <Router>
        <MainContent/>
      </Router>
    </Provider>
  );
}

export default App;
