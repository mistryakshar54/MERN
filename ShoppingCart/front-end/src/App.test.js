import React from 'react';
import {shallow} from 'enzyme';
import App from './App';
import './setupTests';
const app = shallow(<App/>);

it('Renders App component correctly' , ()=>{
    expect(app).toMatchSnapshot();
});
