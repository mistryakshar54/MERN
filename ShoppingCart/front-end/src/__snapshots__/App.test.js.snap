// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders App component correctly 1`] = `ShallowWrapper {}`;
